src/components/Book.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const { book } = props;
+  const { id, title, category } = book;
```

Comment:
can be broken to
```
const { id, title, category } = props;
```

Reply:
It doesn't work like that, because the book itself is an object. The way to do it would be:

`const { id, title, category }  = props.book;`

But linter pops that as an error as well and it wants destructuring from the start.